hrm-service/package-lock.json

```diff
@@ -1,8 +1,12358 @@
 {
   "name": "hrm",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
+12,433 lines because we added the supertest types?? dafuq?

Reply:
This is extremely strange. It looks like it's largely duplicated the contents. Probably has to do with the lockfile version change. @GPaoloni have you recently updated npm versions? I'm on 6.14.15.

Reply:
Yes it is because npm version apparently, I'm in `8.3.2`. Do you want me to rollback this changes in the lockfile?

Reply:
That must be it. I also noticed that Mythily's webchat PR has a regression back to lockfile version 1 - https://github.com/techmatters/webchat/pull/116/files. We should all be on the same major version of NPM -- the question is which one. I will move that conversation to slack.